odoo/sql_db.py

```diff
@@ -763,13 +683,16 @@ def __init__(self, pool, dbname, dsn):
         self.dsn = dsn
         self.__pool = pool
 
-    def cursor(self, serialized=True):
-        cursor_type = serialized and 'serialized ' or ''
+    def cursor(self, **kwargs):
+        if 'serialized' in kwargs:
+            warnings.warn("Since 16.0, 'serialized' parameter is depreciated")
+        cursor_type = kwargs.get('serialized') and 'serialized ' or ''
         _logger.debug('create %scursor to %r', cursor_type, self.dsn)
-        return Cursor(self.__pool, self.dbname, self.dsn, serialized=serialized)
+        return Cursor(self.__pool, self.dbname, self.dsn)
```

Comment:
```suggestion
    def cursor(self, **kwargs):
        if 'serialized' in kwargs:
            warnings.warn("Since 16.0, 'serialized' parameter is ignored", DeprecationWarning, 2)
        assert not (kwargs.keys() - {'serialized'})
        _logger.debug('create cursor to %r', self.dsn)
        return Cursor(self.__pool, self.dbname, self.dsn)
```

Reply:
Why you want ignore the `'serialized'`? I only deprecated it, the behavior should be the same (the log) than before but with a warning.
I am still going to fix the warn and add the assert in both cases.

Reply:
> Why you want ignore the `'serialized'`? I only deprecated it

It's ignored in the Cursor constructor, `serialized` is checked but then 
```diff
-        self.autocommit(False)
+        self.connection.set_isolation_level(ISOLATION_LEVEL_REPEATABLE_READ)
```
I don't really mind, but I figure if that's what we're doing then the warning should say it outright.